pages/research_clusters_and_infra.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Research Clusters & Infrastructures
+---
+
+This section lists research clusters and infrastructures where software is a core part of activities.  
+
+
+{% include section-navigation-tiles.html type="research_cluster_or_infra" search=true col=3 %}
```

Comment:
```suggestion
{% include section-navigation-tiles.html type="research_clusters_and_infrastructures" search=true col=3 %}
```

Reply:
The type of the pages is a research cluster or infrastructure - it's not really research clusters and infrastructures - well it could be but it's less clear I think.